Mathlib/Tactic/TryThis.lean

```diff
@@ -47,13 +47,7 @@ def addRewriteSuggestion (ref : Syntax) (rules : List (Expr × Bool))
     let loc ← loc?.mapM fun loc => do `(location| at $(← delab loc):term)
     `(tactic| rw [$rules_stx,*] $(loc)?)
 
-  let mut tacMsg :=
-    let rulesMsg := MessageData.sbracket <| MessageData.joinSep
-      (rules.map fun ⟨e, symm⟩ => (if symm then "← " else "") ++ m!"{e}") ", "
-    if let some loc := loc? then
-      m!"rw {rulesMsg} at {loc}"
-    else
-      m!"rw {rulesMsg}"
+  let mut tacMsg := m!"{tac}"
```

Comment:
Should this change be reverted in light of [this comment](https://github.com/leanprover/std4/pull/210/files#r1287827045)?